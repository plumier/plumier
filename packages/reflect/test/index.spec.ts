
describe("Exported", () => {
    it("Should export members", () => {
        expect(require("../src")).toMatchSnapshot()
    })
})