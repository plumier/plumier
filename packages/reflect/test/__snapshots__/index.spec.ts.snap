// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exported Should export members 1`] = `
Object {
  "DECORATOR_KEY": "plumier.key:DECORATOR",
  "DESIGN_PARAMETER_TYPE": "design:paramtypes",
  "DESIGN_RETURN_TYPE": "design:returntype",
  "DESIGN_TYPE": "design:type",
  "DecoratorId": Symbol(tinspector:decoratorId),
  "DecoratorOptionId": Symbol(tinspector:decoratorOption),
  "createClass": createClass,
  "decorate": decorate,
  "decorateClass": decorateClass,
  "decorateMethod": decorateMethod,
  "decorateParameter": decorateParameter,
  "decorateProperty": decorateProperty,
  "default": reflect,
  "generic": Object {
    "create": create,
    "template": template,
    "type": type,
  },
  "ignore": ignore,
  "mergeDecorator": mergeDecorator,
  "metadata": Object {
    "getMethods": getMethods,
    "getProperties": getProperties,
    "isCallback": isCallback,
    "isConstructor": isConstructor,
    "isCustomClass": isCustomClass,
    "isParameterProperties": isParameterProperties,
  },
  "noop": noop,
  "parameterProperties": parameterProperties,
  "reflect": reflect,
  "type": type,
  "useCache": useCache,
}
`;
